src/quantum/azext_quantum/operations/workspace.py

```diff
@@ -0,0 +1,78 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=import-outside-toplevel,line-too-long,redefined-builtin
+
+from knack.util import CLIError
+
+from .._client_factory import cf_workspaces
+
+class WorkspaceInfo(object):
+    def __init__(self, cmd, resource_group_name=None, workspace_name=None):
+        from azure.cli.core.commands.client_factory import get_subscription_id
+
+        def select_value(key, value):
```

Comment:
A comment here explaining what this function is doing may be helpful. I figured it out, but it took me a moment as I wasn't familiar with `cmd.cli_ctx`.